articles/Silo_Finance.md

```diff
@@ -0,0 +1,448 @@
+
+**A quick TL;DR of our findings:**
+
+
+
+* Silo Finance introduces a new money market design with isolated markets (silos) for every unique token. These isolated markets are paired with the same counterparty asset called bridge asset (ETH or XAI). Bridge assets represent a concentrated part of liquidity bridged across all isolated markets which facilitates the onboarding (listing) of any token, especially long-tail assets. The isolation of high-risk assets almost completely removes protocol systemic risk, while bridge assets prevent fractured liquidity and keep the protocol liquid and fluid.
+* Silo allows permissionless listing and parameter customization for each silo through governance. Every newly created silo needs to have a reliable price feed source and starts with default collateral factors for Loan-to-Value (LTV), Liquidation Threshold, and Liquidation Penalty.
+* The Silo team takes security very seriously. The core smart contracts were fully audited by Quantstamp and ABDK and tested by the core team through a formal verification process using Certora Prover.
+* Silo recently introduced a new stablecoin named XAI, which will serve as the second bridge asset alongside ETH. XAI can be minted and burnt by the SiloDAO via a governance vote (using Tally).
+* Silo is very progressive in its endeavor to become a decentralized and trustless protocol. They have already fully transitioned to on-chain governance so that SILO token holders are in full control over most of the protocol's functions.
+
+
+# Silo Finance - An Introduction
+
+
+## Siloed Lending Markets
+
+Silo is a permissionless and non-custodial lending protocol that allows the creation of isolated (siloed) lending markets. The protocol is designed to support two types of asset categories, Unique Tokens and Bridge Assets. Unique tokens have their liquidity isolated in dedicated silos (1 silo for each token), while the bridge asset (currently ETH) is paired with every unique token across all isolated silos.
+
+Inside an isolated market, both assets can be used interchangeably as collateral and loan. In other words, when lending token A, one can only borrow the bridge asset ETH against it. The bridge asset ETH can then be used as collateral in another silo, to borrow token B. Thus, one can borrow any token with any collateral, while isolating the risks related to the collateral token within one pool.
+
+In comparison, 1st generation lending markets like Compound or Aave have all collateral in one pool (shared pool), hence the impact of exploits - e.g. through price manipulation - can be much higher. This results in a few drawbacks (e.g.,  limitation in listing new assets, high parameter maintenance cost, cumbersome collateral [listing process](https://medium.com/primedao/enabling-collateral-in-defi-lending-why-your-favorite-token-might-not-be-listed-yet-a1ef27fd19bb), or low capital efficiency). Silo aims to unlock long-tail assets to be lent and borrowed using their new design.
```

Comment:
You might remind readers of the Cream hacks, perhaps the most rekt gen 1 lending platform
https://rekt.news/cream-rekt-2/

Reply:
Nice one, changed the text accordingly.

In comparison, 1st generation lending markets like Compound or Aave have all collateral in one pool (shared pool), hence the impact of exploits - e.g. through price manipulation - can be much higher. This weakness was exploited by hackers multiple times, the lending platform Cream serves as an unholy [example](https://rekt.news/cream-rekt-2/). Other drawbacks of the original shared-pool design include limitations in listing new assets, high parameter maintenance cost, cumbersome collateral [listing process](https://medium.com/primedao/enabling-collateral-in-defi-lending-why-your-favorite-token-might-not-be-listed-yet-a1ef27fd19bb), and low capital efficiency. Silo aims to improve these systemic risks and drawbacks and unlock long-tail assets to be lent and borrowed.